asysource/R3BAsyChimeraReader.h

```diff
@@ -11,26 +16,28 @@
  * or submit itself to any jurisdiction.                                      *
  ******************************************************************************/
 
+#ifndef R3BASYCHIMERAREADER_H
+#define R3BASYCHIMERAREADER_H
```

Comment:
Here you can delete these two lines because 
**#pragma once**
is enough, see https://en.wikipedia.org/wiki/Pragma_once for more details

You could also use **#pragma once** in the definition of all the .h classes